app/core/migrations/0008_auto_20210608_1552.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.1.12 on 2021-06-08 15:52
```

Comment:
We should squash migrations if/where/when possible, since we haven't yet deployed the solution to the customer's environment.

Reply:
Hey Matt I wanted your thoughts on this -- since the component is deployed to our sandbox, if we squash these like we do the others the next migration will ignore the new models unless we wipe the DB and build it again. I guess re-inputting the data again isn't that bad but what do you think?

Reply:
I agree it does make it a bit cumbersome with respect to existing deployments, but I think that we should keep the migration files as squashed as possible until we get to the point where we have essentially lost full control over deployment of the software.  Agree that re-inputting data is a hassle but we could maybe handle that with the Django seed command?

Reply:
Really the only existing deployment we have is the Deloitte sandbox and maybe development workstations.  When we begin to deploy new environments for automated testing those will be new and will need to be seeded, anyway, in order for the Selenium scripts to properly instrument the system.

Reply:
Makes sense; I'm on board with this, we'll work on a seeding script and keep the migrations in 1 file